test-framework/arquillian/src/test/java/io/quarkus/arquillian/test/SimpleTest.java

```diff
@@ -10,16 +10,19 @@
 
 import org.eclipse.microprofile.config.Config;
 import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.junit.InSequence;
+import org.jboss.arquillian.junit5.ArquillianExtension;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-@RunWith(Arquillian.class)
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ArquillianExtension.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
You rewrote all the tests for JUnit 5. Should we at least duplicate one for JUnit 4 if it's possible to have both around?

Reply:
Unfortunately, unless we do some tricky stuff, we can't instruct the surefire plugin to run both junit 5 and junit 4 tests. 
Anyway, all the tcks modules are using the JUnit 4, so we have both covered already.

Reply:
Yeah, fine by me. Thanks!